mets_adapter/mets_adapter/src/main/scala/uk/ac/wellcome/mets_adapter/IngestUpdate.scala

```diff
@@ -1,14 +1,43 @@
 package uk.ac.wellcome.mets_adapter.models
 
-case class IngestUpdate(space: Space, bag: StubBag)
+case class IngestUpdate(space: String, bagId: String)
```

Comment:
Have we agreed with @kenoir or @alexwlchan that the storage service is going to provide updates in this form? 
As far as I know, the storage service exposes two models: `Ingest` and `Bag`. The `Ingest` format is a superset of the `IngestUpdate` class with the `StubBag` as it was before. So with the slightly more convoluted `StubBag` thingy, the adapter would be able to read `Ingest`s, meaning that the storage service wouldn't need to send information in a format specific to the adapter (I should have written this somewhere, sorry). 
It might be worth checking if this matters